docs/sdk/src/external_resources.rs

```diff
@@ -13,8 +13,6 @@
 //! - [Polkadot Blockchain Academy](https://github.com/Polkadot-Blockchain-Academy)
 //! - [Polkadot Wiki: Build](https://wiki.polkadot.network/docs/build-guide)
 
-// Link References
-// [`templates`]: crate::polkadot_sdk::templates
-
-// Link References
-// [`templates`]: crate::polkadot_sdk::templates
+//! Link References
```

Comment:
```suggestion
```

And this line doesn't have to be here, because what comes after is not rendered.

Reply:
Done fixed it.